### PR TITLE
Copyright update year

### PR DESCRIPTION
**Pull request by Filisko**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -98,7 +98,7 @@ To establish a consistent code quality, please check your code using [PHP_CodeSn
 
 (The MIT License)
 
-Copyright (c) 2014 Dominik Liebler and [contributors](https://github.com/domnikl/DesignPatternsPHP/graphs/contributors)
+Copyright (c) 2016 Dominik Liebler and [contributors](https://github.com/domnikl/DesignPatternsPHP/graphs/contributors)
```
Comment (bocharsky-bw) [inline README.md]: I think we should use `2014 - 2016` value instead

**Comment by Filisko** [comment]
Ok, just updated to 2014 - 2016 as you suggested, and yes, it's better.

**Comment by domnikl** [comment]
Thanks for the fix, but the first commit has been made on 21 Aug 2011, so it should be 2011-2016

**Comment by Filisko** [comment]
@domnikl thanks to you! I learned with this repo. Ok, just corrected that.

**Comment by domnikl** [comment]
Looks good now, merged!

**Pull request merged**